docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,138 @@
+
+Before deploying, please note that the script will deploy to the network that your node URL is for. If using Infura for a Mumbai deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mumbai.infura.io/v3/{projectId}
+```
+
+Run the deployment script with your specific parameters.
+```bash
+node index.js --gasprice 1 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x6883FeB1c131F58C1Cd629289Da3dE0051d2aa0d --expirationTimestamp 1643678287 --collateralPerPair 1000000000000000000 --priceIdentifier ETHUSD --collateralToken 0xd0a1e359811322d97991e03f863a0c30c2cf029c --syntheticName "ETH 9000 USD Call [December 2021]" --syntheticSymbol ETHc9000-1221 --financialProductLibrary 0xc19B7EF43a6eBd393446F401d1eCFac01B181ac0
+```
+## Deploying to Polygon Mainnet
+
+The method to deploy onto mainnet is the exact same as with Mumbai, with a few key items to note. Your `url`, `lspCreatorAddress` and `financialProductLibrary` will need to be amended for use on Polygon mainnet. Since the library addresses are different between the networks. 
+
+All the required Polygon Mainnet addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json) with the most important items in the list for specific launch are:
+
+- `Polygon LongShortPairCreator` 
+- `BinaryOptionLongShortPairFinancialProductLibrary` 
+- `CoveredCallLongShortPairFinancialProductLibrary` 
+- `LinearLongShortPairFinancialProductLibrary` 
+- `RangeBondLongShortPairFinancialProductLibrary` 
+
+Please note that the script will deploy to the network that your node URL is for. If using Infura for a Polygon deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+You can now run the deployment script. From within the `launch-lsp` directory, run:
+```bash
+node index.js --gasprice 20 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x3e665D15425fAee14eEF53B9caaa0762b243911a --expirationTimestamp 1643678287 --collateralPerPair 1000000000000000000 --priceIdentifier ETHUSD --collateralToken 0xd0a1e359811322d97991e03f863a0c30c2cf029c --syntheticName "ETH 9000 USD Call [December 2021]" --syntheticSymbol ETHc9000-1221 --financialProductLibrary 0x3F62D7F4Be7671cc93BCDFE7A3Dd900FEC4b5025
+```
+Once deployed, the script will list the address of your newly deployed LSP. A successful output will look like this:
+
+```bash
+Simulating Deployment...
+Simulation successful. Expected Address: 0x44978157afE92c926619EBB54599bbc483eBe871
+``` 
+## Linking your Financial Product Library
```

Comment:
Maybe a better title is: `Setting up your Financial Product Library`, since technically the linkage has already happened